src/screens/Inventory/AddInventory.tsx

```diff
@@ -148,7 +152,7 @@ function AddInventory(props: AddInventoryProps) {
             <div style={{ marginRight: 8, flex: 2 }}>
               <TextField
                 required
-                label={'New Item Name'}
+                label={intl(words.unit_name)}
                 id={'newProductName'}
                 value={formik.values.newProductName}
```

Comment:
This might also need to be run through the translator since it shows the value of `newProductName`

Reply:
What does `newProductName` represent? If this is some variable user input and not a predefined label, then it wouldn't be supported by our translator.

Reply:
Ah ok I understand, yes you're right, thanks!